### DISCUSSION TITLE
Colab price increase?

### DISCUSSION THREAD

**Discussion by SorenTruelsen** [TheLastBen/fast-stable-diffusion#1549] [General]
Hi Guys
The last couple of months I have paid around 50 euro for 500 units. Today I logged on to fill the tank and the price is 450 euro for 500 units. Anyone else noticed this? And .. if this is true ... curse me for cancelling my subscription!

**Comment by TheLastBen** [comment]
450eur for 500 units is unrealistic, check again

**Comment by SorenTruelsen** [comment]
![05E42B72-3C37-4ADE-851F-02C813E198F2](https://user-images.githubusercontent.com/7306254/217161302-3456a57c-540a-4095-b4a7-270e4bcf5e2f.png)

**Reply by tman1980** [reply]
That must be some mistake i just bought it for 9€ without VAT so could be something locally? turn on VPN and check if it changes?

**Comment by TheLastBen** [comment]
In the next few days, I will publish Paperspace Notebooks for Dreambooth and A1111, they offer better plans, save that money for Paperspace.
